game-core/src/main/java/games/strategy/triplea/ui/MapRouteDrawer.java

```diff
@@ -314,28 +323,81 @@ private void drawCurvedPath(final Graphics2D graphics, final Point2D[] points) {
   }
 
   /**
-   * This draws how many moves are left on the given {@linkplain BufferedImage}
-   *
-   * @param image The Image to be drawn on
-   * @param curMovement How many territories the unit traveled so far
-   * @param maxMovement How many territories is allowed to travel. Is empty when the unit traveled too far
+   * This draws current moves, max moves, and fuel cost.
    */
-  private static void createMovementLeftImage(final BufferedImage image, final String curMovement,
-      final String maxMovement) {
-    final Graphics2D textG2D = image.createGraphics();
-    textG2D.setColor(Color.YELLOW);
-    textG2D.setFont(new Font("Dialog", Font.BOLD, 20));
-    final int textThicknessOffset = textG2D.getFontMetrics().stringWidth(curMovement) / 2;
-    final boolean distanceTooBig = maxMovement.isEmpty();
-    textG2D.drawString(
+  private static BufferedImage createMovementLeftImage(final String curMovement, final String maxMovement,
+      final ResourceCollection movementFuelCost, final ResourceImageFactory resourceImageFactory) {
+
+    // Create and configure image
+    final String unitMovementLeft = (maxMovement == null || maxMovement.trim().length() == 0) ? "" : "/" + maxMovement;
```

Comment:
I'm not sure if `" / "` is a better separator than `"/"` here.  The latter seems a bit too cramped to me, but some may think the former is too spread out. :man_shrugging:  Regardless, it's better than the old separator:

```
"    /"
```

Reply:
Not that important, but maxMovement should really be an int with a value of -1 or something for the null replacement if necessary

Reply:
I tried that as well but felt without spaces was more coherent and tied the current moves to max moves better. Yeah, some of the string parameters are a bit interesting but I decided I didn't want to start messing with a bunch of types as they are passed through down to this level.